util/task.py

```diff
@@ -200,9 +200,9 @@ def write_baselines(session):
 
 
 def write_base_values_or_pkv(session):
-    """ If yesterday was the start of the support year, write yesterday's
-    base the base values for all users to the SQLite database.
-    Otherwise, write yesterday's pkv for all users to the QLite database.
+    """ If yesterday was the start of the support year, write the base values
+    for all users to the SQLite database.
+    Otherwise, write yesterday's pkv for all users to the SQLite database.
```

Comment:
```suggestion
    Otherwise, write yesterday's pkv for all users to the SQLite database.
    :param sqlalchemy.orm.scoping.scoped_session session: the database session
```